pkg/storage/couchdb/couchdbstore_test.go

```diff
@@ -122,7 +125,7 @@ func TestNewProvider(t *testing.T) {
 	})
 	t.Run("Blank URL provided", func(t *testing.T) {
 		provider, err := NewProvider("")
-		require.Equal(t, blankHostErrMsg, err.Error())
+		require.Equal(t, errBlankHost, err)
```

Comment:
use `require.EqualError()` instead (third argument is the expected error string, ie `errBlankHost.Error()`)

Reply:
This one is directly checked to see the error objects are fully equal. Is require.EqualError() better to use?

Reply:
You don't need to worry about err.Error() panicking if err is nil.. so yeah, `EqualError()` is the preferred way to check for errors (if you have the full error string only).

Reply:
Good point! I will change it to EqualError

Reply:
Done